packages/plasma-new-hope/src/components/Slider/Slider.tokens.ts

```diff
@@ -27,21 +37,23 @@ export const tokens = {
     labelLetterSpacing: '--plasma-slider-label-letter-spacing',
     labelLineHeight: '--plasma-slider-label-line-height',
 
-    rangeMinValueMargin: '--plasma-slider-range-min-value-margin',
-    rangeMaxValueMargin: '--plasma-slider-range-max-value-margin',
     rangeValueBottomOffset: '--plasma-slider-range-value-bottom-offset',
+    rangeValueHorizontalMargin: '--plasma-slider-range-value-horizontal-margin',
+    rangeValueLeftRightMargin: '--plasma-slider-range-value-left-right-margin',
```

Comment:
Мне смутило, что `LeftRight` по смыслу похоже на слово `Horizontal`, но при этом `rangeValueLeftRightMargin` используется только в вертикальной ориентации. 

Может попробовать по-другому назвать? Например, так:
```suggestion
    rangeValueHorizontalMargin: '--plasma-slider-range-value-horizontal-margin',
    rangeValueVerticalOffset: '--plasma-slider-range-value-vertical-offset',
```

Reply:
можно и на rangeValueHorizontalMargin поменять, сделал rangeValueLeftRightMargin чтоб он с горизонтальным слайдером, наоборот не ассоциировался )